packages/core/src/components/TextField/Mask.jsx

```diff
@@ -104,12 +105,30 @@ Style guide: components.masked-field.react
  * field is blurred, it applies formatting to improve the readability
```

Comment:
Small thing, could you update the comment here to not say `controlled`?

Reply:
I just noticed 94 is outdated, `unmaskValue` should be `unmask` I think.

Reply:
> A Mask component renders a controlled input field
is still a true statement, because it always uses its internal state to provide a value to the input itself. `Mask` is kind of a confusing component because it is a child of the `TextField` and yet it renders the actual HTML `input` element. And this input element is always controlled by `Mask` if `Mask` is rendered.

Reply:
And line 94 is also still accurate because it's exported as `unmaskValue` here: https://github.com/CMSgov/design-system/blob/master/packages/core/src/components/TextField/TextField.jsx#L8-L9

Reply:
Thanks for paying attention to these details though

Reply:
gotcha thanks for clarifying! 👍